src/plugin/minimap/index.js

```diff
@@ -307,35 +307,46 @@ export default class MinimapPlugin {
         }
 
         if (this.params.showOverview) {
-            this.overviewRegion.addEventListener('mousedown', event => {
+            let clientX = (event)=> event.changedTouches ? event.changedTouches[0].clientX : event.clientX;
+            let clientY = (event)=> event.changedTouches ? event.changedTouches[0].clientY : event.clientY;
+            let layerX = (event)=> clientX(event) - event.target.getBoundingClientRect().x;
+
+            let mouseDown = event => {
                 this.draggingOverview = true;
-                relativePositionX = event.layerX;
-                positionMouseDown.clientX = event.clientX;
-                positionMouseDown.clientY = event.clientY;
-            });
+                relativePositionX = layerX(event);
+                positionMouseDown.clientX = clientX(event);
+                positionMouseDown.clientY = clientY(event);
+            };
+            this.overviewRegion.addEventListener('mousedown', mouseDown);
```

Comment:
these event listeners should be cleaned up as well.

Reply:
Makes sense. What is the best way to do that? How are the existing mousedown/mousemove/mouseup cleaned up?

Reply:
take a look at other plugins like regions or timeline.

Reply:
and others in `destroy`: https://github.com/katspaugh/wavesurfer.js/blob/bfa9c76040058ac59bab19e7bd9d347521c6e425/src/plugin/minimap/index.js#L181